pkg/features/kube_features.go

```diff
@@ -723,6 +723,12 @@ const (
 	//
 	// Allows jobs to be created in the suspended state.
 	SuspendJob featuregate.Feature = "SuspendJob"
+
+	// owner: @marosset
+	// alpha: v1.21
+	//
+	// Enables scheduling of 'HostProcess' containers on Windows nodes
```

Comment:
I think the feature supports more than simply "scheduling" such containers, right?

Reply:
Does `Enables support for 'HostProcess' containers on Windows nodes` sound better?